Control/lib/controllers/Run.controller.js

```diff
@@ -61,6 +62,23 @@ class RunController {
       updateExpressResponseFromNativeError(res, error);
     }
   }
+
+  /**
+   * API - GET endpoint to request a re-init of the run service which will update calibration configurations
+   * The handler also must check that the user making the request is allowed for such operation
+   * @param {Request} _ - HTTP Request object
+   * @param {Response} res - HTTP Response object
+   * @returns {void}
+   */
+  async refreshCalibrationRunsConfigurationHandler(_, res) {
+    try {
+      await this._runService.load();
```

Comment:
The purpose of the controller here is to handle the request of an admin/global user that requests the reload of the configuration which is covered by the `load` function. 
What would you propose otherwise?

Reply:
I would say that the service should load what it needs when you call its functions

Reply:
The service already loads what it needs from the in-memory service. 
But there are very rare occurrences when `admins` want to refresh a configuration that was changed in Consul. That is why we need to `re-load` the configuration